src/allmydata/_auto_deps.py

```diff
@@ -104,6 +104,9 @@
     # won't end up stuck on Eliot 1.6.100 with a critical fix only present in
     # 1.6.101.  And if we do, I know how to deal with that situation.
     "eliot >= 1.6.0, <= 1.6.100",
+
+    # A great way to define types of values.
+    "attrs >= 18.2.0",
```

Comment:
Note that `attrs` is not truly a new dependency.  It is already a transitive dependency of Tahoe-LAFS.

Reply:
:+1: